techniques/general/G181.html

```diff
@@ -51,83 +49,80 @@
-         <li>A user had logged in to a secure shopping site and fills out some of the information on an order form. For security reasons, the session times out after 30 minutes, but the user does not submit the form until 45 minutes after loading the page. The user is informed of the time out and is prompted to log-in again. If the user logs in correctly, the order form is presented to the user with all of the data previously entered and the user is able to review their submission and submit the form. If the log-in is not successfully completed, then the form data is discarded by the server.</li>
+         <li>ある Wiki に利用者がログインし、ページの編集を始めた。編集の完了にかかる時間は、サーバーから許可されている不操作セッション時間よりも長いとする。利用者が編集データを送信したが、セッションの時間切れが通知され、ログインページにリダイレクトされた。フォーム送信を処理するスクリプトは、ログインページに元の編集データを変数として受け渡す。利用者のログインが成功すれば、編集画面に戻ることができ、まるでセッションの時間切れがなかったかのようにスクリプトが編集データを処理する。</li>
+         <li>セキュリティが保護されたショッピングサイトに利用者がログインし、注文フォームに情報を入力している。セキュリティ上の理由から、セッションは 30 分後に時間切れとなるが、その利用者はページを読み込んでから 45 分経ってもまだフォームデータを送信していない。利用者には時間切れが通知され、再ログインが促された。利用者が正しくログインできれば、以前に入力したデータが全て残った注文フォームが表示され、内容を確認し、送信できる。ログインが成功しなければ、サーバーによってフォームデータが破棄される。</li>
       </ul>
    </section>
 
-<section id="related"><h2>Related Techniques</h2><ul>
+<section id="related"><h2>関連テクニック</h2><ul>
       <li><a href="../general/G105">G105: Saving data so that it can be used after a user re-authenticates</a></li>
    </ul></section>
-<section id="tests"><h2>Tests</h2>
-      <section class="procedure" id="procedure"><h3>Procedure</h3>
-         <p>On a site that requires user login to submit data:</p>
+<section id="tests"><h2>検証</h2>
+      <section class="procedure" id="procedure"><h3>手順</h3>
+         <p>データの送信にあたって利用者にログインを求めるサイトについて:</p>
          <ol>
-            <li>Log in and begin the timed activity.</li>
-            <li>Allow the session to time out.</li>
-            <li>Submit the data.</li>
-            <li>Re-authenticate.</li>
-            <li>Check that the process can continue and be completed without loss of data, including the original data and any changes made after re-authentication.</li>
-            <li>Check that the process used to save the information submitted in step 3 is not stored on the server. (Note: This requires knowledge of the technology and features used to implement the technique.)</li>
+            <li>ログインし、時間制限のある操作を始める。</li>
+            <li>セッションを時間切れにさせる。</li>
+            <li>データを送信する。</li>
+            <li>再認証する。</li>
+            <li>再認証後に元のデータ及び変更内容が保持された状態で、その後のプロセスが継続可能で、データの欠落なく完了できることを確認する。</li>
```

Comment:
check that ... の訳を、他のテクニックと合わせました。
```suggestion
            <li>再認証後に元のデータ及び変更内容が保持された状態で、その後のプロセスが継続可能で、データの欠落なく完了できることをチェックする。</li>
```